beacon-chain/blockchain/fork_choice.go

```diff
@@ -148,14 +149,28 @@ func (c *ChainService) ApplyForkChoiceRule(
 	if err != nil {
 		return fmt.Errorf("could not hash head block: %v", err)
 	}
+
+	chainHead, err := c.beaconDB.ChainHead()
```

Comment:
```suggestion
	head, err := c.beaconDB.ChainHead()
```
prefer `head` or `blockHead` here

Reply:
there would be two heads here then, one from lmdGhost and the other from `ChainHead()`

Reply:
You are right, that was a bad suggestion